sdk/storage/storage-file-datalake/test/node/highlevel.node.spec.ts

```diff
@@ -38,7 +40,11 @@ describe("Highlevel Node.js only", () => {
 
   beforeEach(async function() {
     recorder = record(this, recorderEnvSetup);
-    const serviceClient = getDataLakeServiceClient();
+    const serviceClient = getGenericDataLakeServiceClient("DFS_", undefined, {
```

Comment:
>getGenericDataLakeServiceClient [](start = 26, length = 31)

Add pipeline options to `getDataLakeServiceClient` as optional parameter. And use `getDataLakeServiceClient({keepAliveOptions: {enabled: true}})` 

We don't want to use `getGenericDataLakeServiceClient` in UTs, as you can see, it duplicates "DFS_" prefix everywhere

Reply:
Fixed.